cmake/ProjectMPIR.cmake

```diff
@@ -1,8 +1,7 @@
 include(ExternalProject)
-include(GNUInstallDirs)
 
 set(prefix "${CMAKE_BINARY_DIR}/deps")
-set(MPIR_LIBRARY "${prefix}/${CMAKE_INSTALL_LIBDIR}/${CMAKE_STATIC_LIBRARY_PREFIX}mpir${CMAKE_STATIC_LIBRARY_SUFFIX}")
+set(MPIR_LIBRARY "${prefix}/lib/${CMAKE_STATIC_LIBRARY_PREFIX}mpir${CMAKE_STATIC_LIBRARY_SUFFIX}")
```

Comment:
Below, `CMAKE_INSTALL_LIBDIR` is being added to cmake args list, but here it's being removed and hard-coded to `lib`?

Reply:
Yes. I'm asking MPIR build to use `lib` as the value of  `CMAKE_INSTALL_LIBDIR`.

Reply:
Same as here: https://github.com/ethereum/solidity/blob/develop/cmake/jsoncpp.cmake#L32.